pkg/cmd/clients.go

```diff
@@ -48,8 +52,8 @@ func (cc *ClientCmd) ListClientsCmd() *cobra.Command {
 		Short: "gets a list of mobile clients represented in the namespace",
 		Long:  `get clients allows you to get a list of mobile clients that are represented in your namespace.`,
 		Example: `  mobile get clients --namespace=myproject 
-  kubectl plugin mobile get clients
-  oc plugin mobile get clients`,
+  					kubectl plugin mobile get clients
+  					oc plugin mobile get clients`,
```

Comment:
This indentation is preferred I would say but I'd suggest change others to this format for consistency since we are touching the file here anyway